taskboard/client.py

```diff
@@ -0,0 +1,22 @@
+from .exceptions import TaskBoardError
+
+class TaskBoardClient:
+    def __init__(self, backend):
+        if not backend:
+            raise TaskBoardError("backend must be provided")
+        self.backend = backend
+
+    def get_tasks(self, list_id: str):
+        return self.backend.get_tasks(list_id)
```

Comment:
this approach has the backend not typed.
also it has an extra "hop" at runtime.

let's have the interface be abstract and method bodies be '...'?

Reply:
Made an abstract base class that will define the interface.